src/interfaces/swig/Preprocessor.i

```diff
@@ -16,7 +16,6 @@
 %rename(PNorm) CPNorm;
 %rename(RescaleFeatures) CRescaleFeatures;
 
-%rename(DimensionReductionPreprocessor) CDimensionReductionPreprocessor;
```

Comment:
why do we drop `CDimensionReductionPreprocessor`? :)

Reply:
`CDimensionReductionPreprocessor` is dropped from both cpp and interface.
It has fields like m_converter, m_kernel, m_distance but they are not used by subclasses it most cases (e.g PCA). So it is not very helpful as a superclass. And we don't need to wrap a converter with it now

Reply:
okie!